### PR TITLE
Add VMScaleSet ScaleIn Policy

### PR DESCRIPTION
**Pull request by ashintre**
Adds VMScaleSet ScaleIn policy that is supported with the 2019-03-01 header but not included in Swagger

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -7482,6 +7482,27 @@
       },
       "description": "The configuration parameters used while performing a rolling upgrade."
     },
+    "ScaleInPolicy": {
```
Comment (michaeljqzq) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: please fix json format

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -7482,6 +7482,27 @@
...
+      "properties": {
+        "rules": {
+          "type": "array",
+          "items": {
+           "type": "string",
+           "enum": [
+            "Default",
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: NIT: two spaces for indent

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 3]
```diff
@@ -8607,7 +8628,11 @@
         "additionalCapabilities": {
           "$ref": "#/definitions/AdditionalCapabilities",
           "description": "Specifies additional capabilities enabled or disabled on the Virtual Machines in the Virtual Machine Scale Set. For instance: whether the Virtual Machines have the capability to support attaching managed data disks with UltraSSD_LRS storage account type."
-        }
+        },
+		"scaleInPolicy": {
```
Comment (michaeljqzq) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: please fix the fomat, same as below

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (michaeljqzq) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: the subsequent property name `scaleInPolicy` is not consistent with this one which is capitalized.

Comment (ashintre) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: That's how it is for all properties.

**Review by hyonholee — APPROVED** [review]

**Review by michaeljqzq — APPROVED** [review]

**Pull request merged**